_drafts/md_abbreviations.md

```diff
@@ -0,0 +1,59 @@
+*[.ics]: Internet Calendaring and Scheduling (iCalendar)
+*[ANSI]: American National Standards Institute
+*[ASCII]: American Standard Code for Information Interchange
+*[BCC]: Blind Carbon Copy
+*[CC]: Carbon Copy
+*[COPR]: Cool Other Package Repo
+*[CR]: Carriage Return
+*[CSS]: Cascading Style Sheets
+*[DBCS]: Double-Byte Character Set
+*[DNS]: Domain Name System
+*[DSN]: Delivery Status Notification
+*[EOF]: End of File
+*[EOL]: End of Line
+*[FOSS]: Free and Open Source Software
+*[GCC]: GNU Compiler Collection
+*[GDBM]: GNU Data Base Manager
+*[GFM]: GitHub Flavoured Markdown
+*[GPGME]: GnuPG Made Easy
+*[GSSAPI]: Generic Security Services Application Program Interface
+*[HTML]: Hyper Text Markup Language
+*[IDN]: Internationalized Domain Names
+*[IMAP4]: Internet Message Access Protocol, version 4
+*[IMAP]: Internet Message Access Protocol
+*[IRC]: Internet Relay Chat
+*[LDAP]: Lightweight Directory Access Protocol
+*[LF]: Line Feed
+*[LMDB]: Lightning Memory-Mapped Database Manager
+*[Lua]: Scripting Language (from Portuguese, meaning moon)
+*[MBCS]: Multi-Byte Character Set
+*[MIME]: Multipurpose Internet Mail Extensions
+*[MUA]: Mail User Agent
+*[Notmuch]: Not much mail
+*[OBDB]: Oracle Berkeley Database
+*[PGP]: Pretty Good Privacy
+*[POP3]: Post Office Protocol, version 3
+*[POP]: Post Office Protocol
+*[PPA]: Personal Package Archive
+*[QDBM]: Quick Data Base Manager
+*[RFC]: Request for Comments
+*[RTF]: Rich Text Format
+*[SASL]: Simple Authentication and Security Layer
+*[SBCS]: Single-Byte Character Set
+*[SMTP]: Simple Mail Transfer Protocol
+*[SOCKS]: Socket Secure (Internet Protocol)
+*[SSL]: Secure Sockets Layer
+*[TBD]: To Be Defined (only one definition)
+*[TLS]: Transport Layer Security
+*[Tar]: Tape archiver
+*[UCS]: Universal Coded Character Set (Unicode, ISO/IEC)
+*[URL]: Uniform Resource Locator
+*[URLs]: Uniform Resource Locator
+*[UTF-16]: 16-Bit UCS Transformation Format
+*[UTF-32]: 32-Bit UCS Transformation Format
+*[UTF-8]: 8-Bit UCS Transformation Format
+*[UTF]: UCS Transformation Format
+*[YAML]: Yet Another Markup Language
+*[ZIP]: archive file format, meaning &ldquo;move at high speed&rdquo;
```

Comment:
I’m not sure which notation to use for curly quotes (literal or entities) because
simple >`" " ' '`< won’t be transformed in curly ones.